l10n/en/brands.ftl

```diff
@@ -69,9 +69,6 @@
 -brand-name-firefox-devtools = Firefox DevTools
 -brand-name-firefox-lockwise = Firefox Lockwise
 
-# Obsolete brand name
--brand-name-firefox-monitor = Firefox Monitor
```

Comment:
**Issue:** I still see this brand name variable used in some obsolete strings. We should keep this variable around as long as those strings still reference it?

Reply:
Seems all the obsolete strings using this brand name variable are 7+ months old, so I'm pretty sure it's safe to remove them :) i'll work on that now. Thanks for catching this!!! 🦅 👁️